### ISSUE TITLE
Start no planejamento

### ISSUE THREAD

**Issue by pedropolisenso** [riojs-org/new-call-list#1]
[ATENÇÃO]

Há um tempinho atrás eu estava falando com @pedromarins sobre criar uma nova ferramenta, decentemente, para substituir de uma vez por todas o **Call4papers** que não tem nenhum segurança e acaba, às vezes, prejudicando uma boa palestras porque o outro coleguinha tem mais amigos e familiares que o anterior e faz com que a mãe, irmã, avó, tios e sobrinhos votem nele com todas as contas possíveis. Quero saber de vocês o que acham desse projeto?

@riojs-org/blog

**Comment by pedromarins** [comment]
Temos que fazer isso! Também acho o modelo atual muito ruim.

Talvez vincular o voto à conta do github. Que tal?

Em sex, 3 de jul de 2015 às 10:12, Pedro Polisenso notifications@github.com
escreveu:

**Comment by pedropolisenso** [comment]
@pedromarins pode ser sim, mas tem outras series de validações a ser feita, saca? Tipo por que negar votos? autenticar login, criar conta deixar o máximo seguro e confiável, porque negar votos é uma baita de uma sacanagem!

**Comment by pedromarins** [comment]
Concordo totalmente. Desenha o projeto que tu pensou ai, para termos um
start.

Em sex, 3 de jul de 2015 às 10:43, Pedro Polisenso notifications@github.com
escreveu:

**Comment by pedropolisenso** [comment]
Beleza qualquer coisa vamos gerenciando tudo pelo TRELLO <3

**Comment by afonsopacifer** [comment]
@pedropolisenso perfeito mano!
Eu mesmo estava muito bolado com o Call4papers esses dias por conta disso, na verdade ate desisti de enviar talks por la. show mesmo!

**Comment by leaofelipe** [comment]
Excelente idéia @pedropolisenso! Vamos botar isso pra frente!

**Comment by pedropolisenso** [comment]
@riojs-org/blog poderíamos dividir a galera entre os projetos q tal? O time do Blog, o Time do Site e o Time do produto pra subistituir o call4, que tal?

CC: @pedromarins

**Comment by pedromarins** [comment]
@pedropolisenso https://github.com/pedropolisenso Os projetos tem que ter
readme com descrição. As pessoas vão participando do que eles mais
gostarem! Acho que não precisa dividir muito não. Acho que vai ser natural
a participação!

Em qua, 8 de jul de 2015 às 14:20, Pedro Polisenso notifications@github.com
escreveu:

**Comment by pedropolisenso** [comment]
Boa @pedromarins faz sentido!

**Comment by pedromarins** [comment]
Tava pensando nisso aqui. Que tal se fosse assim:

Quando o usuário entra, randomizamos a lista de palestras, sem exibir a quantidade de votos que cada uma recebeu? Ele só sabe a quantidade de votos na página do placar, que para ele acessar precisa ter votado em pelo menos uma palestra. 

Além disso, para votar tem que botar o email ou fazer login. E a cada nova palestra que for submetida mandamos um email no modelo: "Você gostaria de ver a a palestra _titulo_da_palestra_ no _titulo_do_evento_?"

Isso não acaba com a galera do fã-clube, mas pelo menos diminui o efeito de "enviei a palestra primeiro" vou ganhar mais votos. Ou, quanto mais votos eu tenho, vou aparecer em primeiro e ganhar ainda mais votos.

O que acham?

**Comment by pedropolisenso** [comment]
A ideia é bem legal, boa proposta. Podemos tbm no email informar que foi submetida a palestra tal tal tal, saca?

**Comment by andrevvalle** [comment]
Curti essa ideia @pedromarins.
Overview:
- Login com Github/Bitbucket.
- Pagina de votação com lista random sem exibir os votos.
- Pagina "placar", o usuario só consegue acessar essa pagina se ja tiver votado em pelo menos uma palestra.

/\* O que acham em deixa publico quem votou em cada palestra? */
Exemplo:
![screen shot 2015-07-13 at 11 50 31 am](https://cloud.githubusercontent.com/assets/7549986/8652571/b59d8f94-2955-11e5-8f22-a5c1f38598b4.png)

**Comment by pedropolisenso** [comment]
@andrevvalle a ideia não é ruim não, porém a ideia é criar uma ferramenta confiável para o uso de todo o Brasil, saca? imagina se mais ou menos (sei que nao rola esses numeros) 150 votos? teríamos 150 fotos? isso vamos dizer por título, imagina 20 propostas com cada proposta ter entre 80 a 200 votos, saca? Acredito q tenha sim, pois no **call4pappers** dá pra negativar, no nosso não vai rolar essa coisa **ESCROTA DEMAIS**. Mas a ideia é válida. Podemos, também, adicionar comentário para as propostas, pode ser legal. Não?

CC: @riojs-org/blog

**Comment by andrevvalle** [comment]
@pedropolisenso nesse caso ficaria assim:

![screen shot 2015-07-13 at 12 16 44 pm](https://cloud.githubusercontent.com/assets/7549986/8653095/ee0828be-2958-11e5-9f8f-413ff0b527c1.png)

Acho muito bom ter comentários tbm ;)

**Comment by pedropolisenso** [comment]
Ahh agora entendi a ideia @andrevvalle eu que interpretei errado, foi mal. Tipo teríamos um limite né, irado vai ficar show mesmo. Achei que iria carregar todas as imagens. Nice!

**Comment by pedromarins** [comment]
Mandou muito bem @andrevvalle! Essa ideia das fotos é perfeita!

Estava pensando sobre o login com github/bitbucket e cogitando botar facebook/twitter. Mas resumir ao github/bitbucket resolve o problema da galera que pede para os familiares votarem. O simples fato de ter que ter um conta nessas plataformas já melhora o processo!

**Comment by pedropolisenso** [comment]
Exatamente, sem contar que ainda é bom ter login por cadastro né? Pode haver pessoas que não tenham esses 2 sistemas. Ou não?

**Comment by rdiego26** [comment]
Concordo plenamente :)
Em 13/07/2015 1:48 PM, "Pedro Polisenso" notifications@github.com
escreveu:

**Comment by pedromarins** [comment]
Tive mais uma ideia. Que tal se exibirmos a palestra para ser votada sem os dados do palestrantes. Só o título e descrição da palestra. Com isso as pessoas fazem um julgamento sem interferências. O foco é só no conteúdo. Que tal?

**Comment by pedropolisenso** [comment]
@pedromarins eu não sei os outros, mas em um evento o que me interessa é o que significa o que o cara vai falar, saca? um exemplo: No front-end carioca o tema do @rdiego26 é HARDENING NODE, eu sei que é sobre node, mas o que exatamente? Segurança? API's? O Back-end? Server? coisas dessa maneira, com a descrição eu terei uma ideia se me interessa ou nao. Acho que é isso mano =)

**Comment by pedromarins** [comment]
Mas é exatamente isso. Exibirmos o titulo e descrição da talk, sem o dono
dela. O foco é o conteúdo, e não o palestrante.

Em ter, 11 de ago de 2015 às 13:44, Pedro Polisenso <
notifications@github.com> escreveu:

**Comment by pedropolisenso** [comment]
Entendi sua ideia, eu que interpretei errado, porém nao é algo muito certo, pelo fato de que o cara vai pedir pra votarem nele, saca? Essa sua ideia é muito boa, pq evita aqueles que escolhem famosos escolherem apenas famosos, porem tem isso. O cara vai falar: Vota na minha palestra la o tema é X saca? Pode ser nao ser um problema, mas alguns farão isso
